mkdocs.yml

```diff
@@ -145,6 +145,9 @@ nav:
     - Vendoring: "development/vendoring.md"
     - Ports: "development/ports.md"
   - Releases:
+    - 1.19: releases/1.19-NOTES.md
```

Comment:
Perhaps not list 1.19 and 1.18 just yet?

Reply:
Yeah, I wasn't sure about 1.19 since it's not really released yet. 
But since the[ file is already there](https://github.com/kubernetes/kops/blob/master/docs/releases/1.19-NOTES.md) and notes that right now just state that it hasn't been released yet, I figured we _could_ just add it and whoever clicks it will see that it's not released yet.
I'm okay with taking it out if you think we shouldn't include it yet, let me know 😄

Reply:
Remove the 1.19 one. I see that we link to 1.18 from the releases page, but perhaps add "(beta)" to it?

Reply:
Done 😄